app/models/post.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
+  belongs_to :user, foreign_key: 'user_id'
```

Comment:
- [ ] We don't need the foreign key unless you changed the name of the model, like `belongs_to :author, foreign_key: 'user_id'`